src/Maestro/DependencyUpdater/DependencyUpdater.cs

```diff
@@ -79,27 +79,61 @@ public async Task StartUpdateDependenciesAsync(int buildId, int channelId)
         }
 
         /// <summary>
-        ///     Run a single subscription
+        ///     Run a single subscription, only accept the build Id specified 
         /// </summary>
         /// <param name="subscriptionId">Subscription to run the update for.</param>
+        /// <param name="buildId">BAR build id to run the update for</param>
+        /// 
+        /// <returns></returns>
```

Comment:
Why add the blank line and the empty returns tag?

Reply:
removed